proto/sedachain/pubkey/v1/genesis.proto

```diff
@@ -11,6 +11,8 @@ option go_package = "github.com/sedaprotocol/seda-chain/x/pubkey/types";
 message GenesisState {
   repeated ValidatorPubKeys validator_pub_keys = 1
       [ (gogoproto.nullable) = false ];
+  repeated ProvingScheme proving_schemes = 2 [ (gogoproto.nullable) = false ];
+  Params params = 3 [ (gogoproto.nullable) = false ];
```

Comment:
I know that adding new fields should be always at the end to make it not a breaking change. But does it make sense to consider having Params at index 1 as most modules in Cosmos SDK?